runtime/src/append_vec.rs

```diff
@@ -130,7 +134,9 @@ impl AppendVec {
         //crash on some architectures.
         let pos = align_up!(*offset as usize, mem::size_of::<u64>());
         let data = &self.map[pos..(pos + len)];
-        //this mut append is safe because only 1 thread can append at a time
+        //UNSAFE: This mut append is safe because only 1 thread can append at a time
```

Comment:
For the alignment comment above, is it 64-byte or 64-bit/8 byte? For me the safety is also about non-null, length not causing out-of-bounds and pointers being aligned than just thread-safety.